### ISSUE TITLE
Missing dependencies?

### ISSUE THREAD

**Issue by nikjdwhite** [wgcowan/KalturaVideo#1]
A really promising and useful plugin!

I'm attempting to set this up on an Omeka instance at the University of Iowa, and it seems like there are dependencies that are not packaged into the plugin. Some of them, like video-js and jquery-ui, we're familiar with, but others (evia, rbox) we're unsure of and are having a hard time finding. Do you have a list of the dependencies for KalturaVideo and/or recommendations for setting it up?

**Comment by wgcowan** [comment]
I thought I had cleared up those dependencies. I'll take a look and see what's going on.

Will

**Comment by nikjdwhite** [comment]
Thanks!

**Comment by wgcowan** [comment]
All the dependencies are in the plugin. The evia.css and rbox.css files are in the views/public/css/res directory. These css files are only used if you choose the option to display metadata next to the video playback. In addition, for kaltura videos, you need to fill out all the fields in the Kaltura Video tab when you edit an item, replacing the xxxxxxxx with your university’s UI Conf and the yyyyyyyyy with your Partner ID.  These values can also come from Kaltura’s public web site for your individual account. See the example below.

Kaltura Video

Video File ID
1_dji9482v

Video UI Conf   xxxxxxxxx
Partner ID   yyyyyyyyyyy
Segment Start
0

Segment End
600

Segment Type
Scene

Show Item
TRUE

Video Source
Kaltura





In addition, since as far as I know there is no way to log in with the Kaltura API, all videos you stream from Kaltura will need to be public.

I hadn’t quite gotten everything ready for release for this plugin yet. Thanks for downloading and I would be interested in any problems or suggestions you might have. 

Will

William G. Cowan
Green Darner Consulting for Omeka
www.green-darner.com

**Comment by nikjdwhite** [comment]
Thanks for getting back to me! 

It looks like when I unpacked it I skipped some files because a few were throwing errors, and the css folder got lost in the shuffle. We re-downloaded and unpacked the plugin skipping the problem files and it seems to be working correctly! We're running into other things but I'll open that as a new issue for clarity.

The files that threw errors during unzipping were:

- views/public/vidplayer (4_12_18, 02_44).php
- views/public/vidplayer (4_12_18, 11_35).php
- views/shared/exhibit_layouts/kalplayer/layout (4_11_18, 10_14).php
- views/shared/exhibit_layouts/kalplayer/layout (7_8_18, 4_22).php

But excluding them doesn't seem to be causing any problems.
